src/components/Modal/VPDelegationModal.tsx

```diff
@@ -0,0 +1,124 @@
+import './VPDelegationModal.css'
+
+import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
+import Avatar from 'decentraland-gatsby/dist/components/User/Avatar'
+import useFormatMessage, { useIntl } from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { navigate } from 'decentraland-gatsby/dist/plugins/intl'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Close } from 'decentraland-ui/dist/components/Close/Close'
+import { Modal, ModalProps } from 'decentraland-ui/dist/components/Modal/Modal'
+import { Table } from 'decentraland-ui/dist/components/Table/Table'
+import React, { useRef, useMemo, useState } from 'react'
+import { IntlShape } from 'react-intl'
+
+import { snapshotUrl } from '../../entities/Proposal/utils'
+import useVotingPowerBalance from '../../hooks/useVotingPowerBalance'
+import exampleDelegates from '../../modules/delegates/example_delegates.json'
+import locations from '../../modules/locations'
+import Sort from '../Icon/Sort'
+
+type Delegate = {
+  address: string
+  ens_name: string | undefined
+  total_vp: number
+}
+
+type VPDelegationModalProps = Omit<ModalProps, 'children'> & {
+  vp: number
+  space: string
+}
+
+function addressShortener(address: string) {
+  return address.substring(0, 6) + '...' + address.substring(38, 42)
+}
+
+function tableCell(delegate: Delegate, picked_by: number, intl: IntlShape, arrowRef: React.RefObject<HTMLDivElement>) {
+  return (
+    <>
+      <Table.Cell>
+        <span className="Candidate">
+          <Avatar address={delegate.address} size="small" />
+          {delegate.ens_name || addressShortener(delegate.address)}
+        </span>
+      </Table.Cell>
+      <Table.Cell>{intl.formatNumber(picked_by)}</Table.Cell>
+      <Table.Cell>{intl.formatNumber(delegate.total_vp)}</Table.Cell>
+      <Table.Cell>
+        <div ref={arrowRef} className="Arrow" />
+      </Table.Cell>
+    </>
+  )
+}
+
+function VPDelegationModal({ vp, space, ...props }: VPDelegationModalProps) {
```

Comment:
As @1emu said with `dgt`, I'd also avoid it here with `VP` and write `VotingPowerDelegationModal`